conf/test_metaphlan.config

```diff
@@ -0,0 +1,33 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test for
+    metagenomics krakenuniq.
+
+    Use as follows:
+        nextflow run nf-core/eager -profile test_krakenuniq,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Metaphlan3 test profile'
+    config_profile_description = 'Minimal test dataset to check the metagenomics metaphlan3 pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    input = 'https://github.com/nf-core/test-datasets/raw/eager/testdata/Mammoth/samplesheet_v3.tsv'
+
+    // Genome references
+    fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/eager/reference/Mammoth/Mammoth_MT_Krause.fasta'
+
+    // Metagenomics
+    run_metagenomics                = true
+    metagenomics_profiling_tool     = 'metaphlan'
+    metagenomics_profiling_database = 'https://github.com/nf-core/test-datasets/raw/409834b927c3a4e9314691b1125acee1434f7dd8/data/delete_me/metaphlan4_database.tar.gz'
```

Comment:
We should copy this over to the eager branch too for stability (`delete_me` will indeed possibly be deleted soon :grimacing: )

Reply:
done